src/main/kotlin/app/revanced/patches/twitter/layout/hideviews/patch/HideViewsResourcePatch.kt

```diff
@@ -10,7 +10,7 @@ import app.revanced.patcher.patch.annotations.Patch
 import app.revanced.patches.twitter.layout.hideviews.annotations.HideViewsCompatibility
 import org.w3c.dom.Element
 
-@Patch
+@Patch(false)
```

Comment:
Why is this not recommended?

Reply:
The current motivation to make it excluded by default is only supported by the fact that another patch doesn't support that version, but patches shouldn't depend on each other so this motivation isn't backed. When the patch itself is undesired, for example, if it has a side effect or causes unintended behavior then it is recommended to make it not included by default. If the patches are included by default, but the version does not match, the patch will be automatically skipped by the CLI and not included by the ReVanced Manager app.